.github/runtime.json

```diff
@@ -3,7 +3,7 @@
       "name": "rococo",
       "package": "rococo-parachain-runtime",
       "path": "runtime/rococo",
-      "uri": "wss://rpc.rococo-parachain.litentry.io:443"
+      "uri": "wss://rpc.litentry-parachain.litentry.io:443"
```

Comment:
I think this one is not necessary. But the change also doesn't hurt anything. Rococo will be deprecated/removed soon.